core/templates/css/oppia.css

```diff
@@ -461,6 +461,7 @@ h3 {
 a {
   color: #0844aa;
   cursor: pointer;
+  text-decoration: underline;
```

Comment:
This seems to be a sweeping site-wide change. Would it be better to use a CSS modification that's more local to the privacy/terms page instead?

Reply:
Even though I think it might be better to have on all links, to avoid the same a11y issue in the future on other pages, I understand that it might break specific pages, since it's my first contribution on this repo, and therefore I don't have enough understanding on the system, I'll restrain these changes only to the pages where the issue needs to be fixed.

Thanks for the feedback.